extension/src/background/helpers/account.ts

```diff
@@ -155,6 +156,9 @@ export const subscribeAccount = async (publicKey: string) => {
     await localStore.setItem(HAS_ACCOUNT_SUBSCRIPTION, subsByKeyId);
   } catch (e) {
     console.error(e);
+    captureException(
```

Comment:
I'm not actually super sure about what we should do if we fail to subscribe an account. When/if this happens we will end up showing an empty account when in reality we could just be failing to index it. It might be worth thinking about a way to toggle Mercury off when this happens so that we can investigate/fix the missed subscription?

Reply:
This is an interesting thought and related to something I've been thinking about, too. If Mercury/Freighter BE is down (or fails for some reason), should we keep some basic Horizon functionality to fallback to in this extension so users don't think somehow their funds have disappeared when really it was an API error?

Reply:
Well so if Mercury fails in the lifecycle where we fetch their data like account details or history, then the backend falls back to using Horizon or rpc. If the API hard errors, then we show an error screen, but in the case that we fail to subscribe but the Mercury still responds to read queries then we don't have a good story since they would see the empty account in that case.
I think this edge case is complex though, we would likely need to take action when a sub fails and either use Horizon/rpc for the pub key that failed or use it for everything while we investigate the sub failure and restore it.